site-content/source/modules/ROOT/pages/native_protocol.adoc

```diff
@@ -3,21 +3,81 @@
 
 == Native Protocol Version 3
 
-[source, plaintext]
-----
-include::example$TEXT/native_protocol_v3.spec[Version 3]
-----
+[source, js]
+++++
+<div id="contentv3" class="doc-container"></div>
+<iframe id="iframev3" src="native_protocol_v3.html" style="display:none;"></iframe>
```

Comment:
is there a way to include the generated native_protocol_v3.html page, instead of using an iframe ?

Reply:
It can be done using the `fetch` but it's relying on the same `CORS` policy.

Reply:
can it not be included (somehow) at antora build time ?  if possible that would be the best IMO.

Reply:
Could you please clarify - did you mean the JavaScript contained within the .adoc file?

Reply:
no. javascript and iframes are all doing the including at view time.

we should be embedding the html contents into the other file at antora build time.

Reply:
this is what i mean: https://github.com/apache/cassandra-website/commit/c7b6b60eb51c4c27e070adb9f3596e979491eb2f 

note, some small things are still needed on top of that^
 (e.g. there's links in the headers of each page that don't work, and we need gitignore if we have to put the generated files under site-content)

Reply:
Understood. I'll try to move it into the `.hbs` layout.

Reply:
Also… there's another thing we might be cleaned up here.

The native protocol spec files are actually here:
https://github.com/apache/cassandra/tree/trunk/doc

They're the source of truth, and those that will be kept up to date.

IUUC the ones in cassandra-website and unnecessary duplicates: https://github.com/apache/cassandra-website/tree/trunk/site-content/source/modules/ROOT/examples/TEXT

We can assume that these from apache/cassandra:trunk are accurate. And delete the `examples/TEXT` directory.    And it might be possible then to not need any gitignore addition.

An alternative here would be to also remove the native_protocol.adoc file from casandra-website altogether, creating this file instead here: https://github.com/apache/cassandra/tree/trunk/doc/modules/cassandra/pages 
(this would mean running `process-native-protocol-specs-in-docker.sh` on each `generate_cassandra_versioned_docs()`)